src/calendar.tsx

```diff
@@ -81,13 +81,6 @@ export function Calendar({
       })
     }
 
-    if (
-      (value.start && isSameDay(date, value.start)) ||
-      (value.end && isSameDay(date, value.end))
-    ) {
```

Comment:
I think a better approach here should be to add a new prop to the library something like `allowSelectSameDay` with this we don't need to remove this for now.